CHANGES.md

```diff
@@ -35,6 +35,12 @@ To be released.
  -  `Swarm` class now does not implement `IEquatable<Swarm>` anymore and
     its `Equals(object)` method and `GetHashCode()` method became to have
     default behavior of `object` class.
+ -  The type of `Block<T>.Difficulty` is changed to `long` instead of `int`, and
+    related classes method parameters and field types have changed accordingly.
+ -  Removed `HashDigest.HasLeadingZeroBits()` method.  [[#213]]
+ -  Added `HashDigest.Satisfies()` method.  [[#213]]
+ -  `BlockPolicy<T>` constructor became to receive the <q>minimumDifficulty</q>
+    and the mining <q>difficultyBoundDivisor</q>.  [[#213]]
```

Comment:
Shouldn't `minimumDifficulty` and `difficultyBoundDivisor` be backquoted (not wrapped by `<q>` tag)?